utils/aeternity.js

```diff
@@ -241,7 +238,7 @@ class Aeternity {
   }
 
   async getChainNames() {
-    return axios.get(`${MIDDLEWARE_URL}/middleware/names/active`).then(res => res.data).catch(this.logger.error);
+    return this.iterateMdw('names/active?limit=1000');
```

Comment:
Can we pass the limit as function argument?

Reply:
why do we need this? the goal is to fetch all pages, so the limit doesn't matter

Reply:
Why is it there in the first place?

Reply:
without limit you get 10 items per page, 1000 is the maximum limit

Reply:
Ok. So you iterate over the next 1000 (or what is left) in the iterateMdw fn. clear now